migrations/4_stablecoin_converter.js

```diff
@@ -0,0 +1,23 @@
+/*eslint no-undef: "off"*/
+
+const BiMap = artifacts.require("IdToAddressBiMap.sol")
+const IterableAppendOnlySet = artifacts.require("IterableAppendOnlySet.sol")
+const StablecoinConverter = artifacts.require("StablecoinConverter.sol")
+const ERC20Mintable = artifacts.require("ERC20Mintable.sol")
+
+module.exports = async function (deployer, network) {
+  let fee_token
+  if (network == "development" || network == "coverage") {
+    await deployer.deploy(ERC20Mintable)
+    fee_token = await ERC20Mintable.deployed()
+  } else {
+    throw (`No migration found for network "${network}"`)
+  }
+
+  await deployer.deploy(BiMap)
+  await deployer.deploy(IterableAppendOnlySet)
+
+  await deployer.link(BiMap, StablecoinConverter)
+  await deployer.link(IterableAppendOnlySet, StablecoinConverter)
+  await deployer.deploy(StablecoinConverter, 2 ** 16 - 1, 1000, fee_token.address)
```

Comment:
Shouldn't be the fee token provided instead of deployed as a new token? 
Maybe this is related to the description that this is for local development.

One idea is to define for example a `getFeeTokenAddress` function, that will deploy the ERC20 and return the address, and throws an exception if any other case, saying that network XXX is not supported.

Reply:
> Maybe this is related to the description that this is for local development.

Yes, once @josojo writes the rinkeby deployment he will use the already deployed GNO contract instead of deploying a new mintable token.

I think the idea of extracting the logic of deriving the fee token address into its own method is nice.

Reply:
Yep, the advantage is that it crashes if you try to deploy to a network you shouldn't.

Maybe the name I suggested is not very good. Is not a good pattern to give a `getter`name to sth that deploys a contract, but with a better name would be fine, hehe

Reply:
It should crash at the moment when you try to deploy to a network that is not supported. I'm throwing in the else branch above.